doc/installation.rst

```diff
@@ -113,14 +139,14 @@ Alternative methods
 .. caution::
 
    While we offer alternative Spyder installation options for users who desire them, we currently lack the resources to offer individual assistance for problems specific to installing via these alternative distributions.
-   Therefore, we recommend you switch to Anaconda if you encounter installation issues you are unable to solve on your own.
+   Therefore, we recommend you switch to our :ref:`standalone installers<standalone_installers_ref>` (Windows and macOS) or :ref:`anaconda_install_ref` if you encounter installation issues you are unable to solve on your own.
 
 
 Windows
 ~~~~~~~
 
-Spyder is also included in the `WinPython`_ scientific Python distribution, although it doesn't include Anaconda's convenient ``conda`` package and environment manager like Anaconda.
-You can use it immediately after installing, just like with Anaconda.
+Spyder is included in the `WinPython`_ scientific Python distribution (although Anaconda's powerful ``conda`` package and environment manager is not).
```

Comment:
Not sure If the "powerful" is necessary. Sounds a bit pretentious and it doesn't add any important information.